Sources/TripKit/model/TKCoordinates.swift

```diff
@@ -124,6 +127,12 @@ public class TKStopCoordinate: TKModeCoordinate {
       availableRoutes = try values.decodeIfPresent(Int.self, forKey: .availableRoutes)
       routes = try values.decodeIfPresent([TKAPI.Route].self, forKey: .routes)
       operators = try values.decodeIfPresent([TKAPI.Operator].self, forKey: .operators)
+      
+      if let rawAccessibility = try values.decodeIfPresent(Int.self, forKey: .popularity) {
+        wheelchairAccessibility = .init(rawValue: rawAccessibility) ?? .unknown
+      } else {
+        wheelchairAccessibility = .unknown
```

Comment:
is there a particular use case for `wheelchairAccessibility` being `nil`?

Reply:
No, only backwards compatibility when trying to load such an object before it was written the new way, e.g., from CoreData.